tests/draw.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { MemoryModel } from "../src/memory_model";
+const { draw } = require("../dist/memory_models_rough");
+
+describe("draw function", () => {
+    let objects: Array<object>;
+    beforeEach(() => {
```

Comment:
I wouldn't use a `beforeEach` here. For now you can define the variable directly in the test case.

For generalizing, what I'd want to see is multiple test cases, each with their own snapshot test _with a different input JSON for each test_.